DigitalOffice/Services/ProjectService/Repositories/ProjectRepository.cs

```diff
@@ -53,5 +54,32 @@ public Guid EditProjectById(DbProject dbProject)
 
             return dbProject.Id;
         }
+
+        public void DisableWorkersInProject(WorkersIdsInProjectRequest request)
+        {
+            DbProject dbProject = dbContext.Projects
+                .FirstOrDefault(p => p.Id == request.ProjectId);
+
+            if (dbProject == null)
+            {
+                throw new NullReferenceException("Project with this Id does not exist");
+            }
+
+            foreach (Guid workerId in request.WorkersIds)
+            {
+                DbProjectWorkerUser dbProjectWorker = dbProject.WorkersUsersIds?
+                    .FirstOrDefault(w => w.WorkerUserId == workerId);
+
+                if (dbProjectWorker == null)
+                {
+                    throw new NullReferenceException("Worker with this Id does not exist");
+                }
```

Comment:
I agree that we should throw if a worker does not exist, but what if only one Guid messed up along the way and out of 10 workers there's only one that does not exist?
Should we really throw the exception or disable all of them that we can?
#discuss